packages/client-core/src/world/Location.tsx

```diff
@@ -38,11 +38,15 @@ import './LocationModule'
 
 import { t } from 'i18next'
 
+import multiLogger from '@etherealengine/common/src/logger'
 import { StyledEngineProvider } from '@mui/material/styles'
 import { LoadingCircle } from '../components/LoadingCircle'
 import { useLoadEngineWithScene, useNetwork } from '../components/World/EngineHooks'
 import { LocationService } from '../social/services/LocationService'
 import { LoadingUISystemState } from '../systems/LoadingUISystem'
+import { clientContextParams } from '../util/contextParams'
+
+const logger = multiLogger.child({ component: 'system:location ', modifier: clientContextParams })
```

Comment:
Just because I noticed while managing weird conflict errors when updating MT-INT. The component string has a space at the end here.

Reply:
I fixed this, and another one in https://github.com/EtherealEngine/etherealengine/pull/10759/files